app/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -376,13 +376,35 @@ describe("Summary", () => {
     const endExperimentButton = await screen.findByTestId(
       "end-experiment-start",
     );
-    const endEnrollmentButton = await screen.findByTestId(
-      "end-enrollment-start",
-    );
     await waitFor(() => {
       expect(requestUpdateButton).toBeInTheDocument();
-      expect(endEnrollmentButton).toBeInTheDocument();
       expect(endExperimentButton).toBeInTheDocument();
     });
   });
+
+  it("do not show end enrollment button for live dirty rollout", async () => {
```

Comment:
Should be for a rollout in any state here

Reply:
The enrollment end button is only shown when live, so it is only hidden for live rollouts